test/unit-test/FreeRTOS_IP_DiffConfig/FreeRTOS_IP_DiffConfig_utest.c

```diff
@@ -257,57 +257,57 @@ static BaseType_t StubxTaskCreate( TaskFunction_t pxTaskCode,
     return pdPASS;
 }
 
-void test_FreeRTOS_IPInit_HappyPath( void )
-{
-    const uint8_t ucIPAddress[ ipIP_ADDRESS_LENGTH_BYTES ] = { 0xC0, 0xB0, 0xAB, 0x12 };
-    const uint8_t ucNetMask[ ipIP_ADDRESS_LENGTH_BYTES ] = { 0xC1, 0xB2, 0xAC, 0x13 };
-    const uint8_t ucGatewayAddress[ ipIP_ADDRESS_LENGTH_BYTES ] = { 0xC2, 0xB3, 0xAC, 0x14 };
-    const uint8_t ucDNSServerAddress[ ipIP_ADDRESS_LENGTH_BYTES ] = { 0xC3, 0xB4, 0xAD, 0x15 };
-    const uint8_t ucMACAddress[ ipMAC_ADDRESS_LENGTH_BYTES ] = { 0xAA, 0xBB, 0xCC, 0xDD, 0xEE, 0xFF };
-    BaseType_t xReturn;
-    QueueHandle_t ulPointerToQueue = ( QueueHandle_t ) 0x1234ABCD;
+// void test_FreeRTOS_IPInit_HappyPath( void )
```

Comment:
why is this one removed? this means everything working correctly right? Does we not need similar test case?

Reply:
this particular tests IPConfig file disables `ipconfigCOMPATIBLE_WITH_SINGLE` hence `FreeRTOS_IPInit` wont be available, so the test is moved to other diffconfig1 and diffconfig2